tests/Agent/IntegrationTests/IntegrationTestHelpers/RemoteServiceFixtures/RemoteService.cs

```diff
@@ -114,6 +114,9 @@ private void PublishWithDotnetExe(string framework)
 
             TestLogger?.WriteLine($"[RemoteService]: Publishing to {deployPath}.");
 
+            var sw = new Stopwatch();
```

Comment:
Is this supposed to be part of this PR?

Reply:
Yeah, I threw it in when trying to investigate where time was being spent, it seemed like the publish step was slow sometimes.